src/SFA.DAS.SharedOuterApi/Models/Application.cs

```diff
@@ -24,12 +24,16 @@ public class Application
         public Standard Standard { get; set; }
         public bool IsNamePublic { get; set; }
         public string Status { get; set; }
+        public IEnumerable<string> PledgeLocations { get; set; }
```

Comment:
It feels as though too much is being added to the application object coming back from the inner api - perhaps a separate call to get the pledge would be better?